components/brave_federated_learning/brave_federated_learning_features.cc

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_federated_learning/brave_federated_learning_features.h"
```

Comment:
@moritzhaller should we maybe consolidate `brave_federated_learning` and `brave_operation_patterns` feature files into a single one?

Reply:
sgtm